src/types.ts

```diff
@@ -73,6 +73,7 @@ export interface ContentfulBlogPost {
   introText?: {
     introText: string;
   };
+  introRichText?: RenderRichTextData<ContentfulRichTextGatsbyReference>;
```

Comment:
The type is wrong here, as it's not really all the rich text fields but only `raw`. Can you please align that and also add a comment on why this is (currently) necessary?

Also, the `introText` above can be removed, right?

Reply:
I removed the introText and changed the type, but to fit the RichText Props I had to do this ugly thing:

https://github.com/satellytes/satellytes.com/blob/bcf99dd66afddf1ea685562cecce0dfc68ad714a/src/components/pages/blog-post/blog-post.tsx#L82-L87

Have you any idea how I could avoid this? The types come directly from gatsby, so I cannot make references optional.